examples/spark/src/main/scala/org/apache/carbondata/examples/CaseClassDataFrameAPIExample.scala

```diff
@@ -42,8 +45,25 @@ object CaseClassDataFrameAPIExample {
       .mode(SaveMode.Overwrite)
       .save()
 
+    val employee = List(Employee("anubhav", 20000))
```

Comment:
I change the code as suggested in code snippet,earlier I am not giving precision and scale factor,ya sure I am adding test cases